### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_MenuBar_ScreenReader: Narrator is not announcing "state of links"(enable or disable) under Edit tab.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#3958]
**EnvironmentDetails:**
Application Name:AzureNotebooks_Jyupter
Login Url: notebooks.azure.com
BrowserDetails:
Edge:Microsoft Edge 41.16299.611.0
Operating System:Windows10

**ReproSteps:**
1. Login with the appropriate credentials to azure notebook account.    
2. Navigate to any of the file names present in the samples-->>click on the file name-->Welcome page opens with menubar.                                           
3. Start Narrator.                                                                                 
4.Navigate to Edit tab Links.                                                                              

**Actual:**
Narrator in not announcing "state of links"(enable or disable) under Edit tab.

**Note:**
Same issue is observed in **NVDA** also.
Same issues observed for all the links which are in Enable or Disable State present under all Menus.

**Expected:**
Narrator should announce "state of links"(enable or disable) under Edit tab.

**User Impact:**
Difficult to understand the current state of the link for Screenreader users.

**Attachment:**
(https://github.com/jupyter/notebook/files/2399942/A11y_AzureNotebooks_Jupyter_MenuBar_ScreenReader.pptx)

**Comment by jtpio** [comment]
Closing as there are now different issues tracking accessibility improvements for Jupyter Notebook 7, for example https://github.com/jupyter/notebook/issues/6800

Thanks!
